src/EpsilonDashboard.pro

```diff
@@ -17,3 +17,5 @@ SUBDIRS = \
 DISTFILES += \
     .travis.yml \
     config.ini
+
+system(cd $$OUT_PWD; conan install .. --build=outdated -s compiler.libcxx="libstdc++11")
```

Comment:
I'm considering have this step be manual, for cross compiling, I think I would want them to use a separate profile with the RPI's information. To do that, the command would be `conan install .. <blah flags> -p=<path to profile>`. If we want to have support for different profiles then having toned flags to make would be unwieldy.

The other thing is is that conan docs such as the [getting started tutorial](https://docs.conan.io/en/latest/getting_started.html#an-md5-hash-calculator-using-the-poco-libraries)  have the pattern of using `conan install`, then `generator build` as separate steps, so I think we should do that too, instead of coupling it together in qt, which might cause troubleshooting issues later for maintainers. What are your thoughts?